lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -121,6 +121,17 @@ function dosomething_mbp_update_7006(&$sandbox) {
 }
 
 /**
+ * Updates user_register production to:
+ * - include imageProcessingQueue
+ * - remove userCampaignActivityQueue
+ * from transactional_campaign_reportback production.
+ */
+function dosomething_mbp_update_7007(&$sandbox) {
+  db_drop_table('message_broker_producer_productions');
+  dosomething_mbp_install_productions();
```

Comment:
If it's worked for other deploys, then no biggie.  I just meant using `db_delete` and `db_insert` instead of completely dropping the table.

While we're in here, might as well mention I got these errors upon running a `ds build --install`:

````
Missing argument 1 for dosomething_mbp_update_7006(), called in      [warning]
/var/www/dev.dosomething.org/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install
on line 29 and defined in dosomething_mbp_update_7006() (line 113 of
/var/www/dev.dosomething.org/lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install).
````

Probably not critical if everything's okl, but, seems somewhat relevant.  Would be cleaner to not get those errors upon install.